dotCMS/src/main/java/com/dotmarketing/portlets/containers/model/ContainerView.java

```diff
@@ -0,0 +1,44 @@
+package com.dotmarketing.portlets.containers.model;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.portlets.containers.business.FileAssetContainerUtil;
+import com.dotmarketing.portlets.htmlpageasset.business.render.page.PageViewSerializer;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.io.CharArrayReader;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+@JsonSerialize(using = ContainerViewSerializer.class)
+public class ContainerView {
+    private Container container;
+    private Host host;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'host' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)